frontend/src/lib/components/expander/Expander.svelte

```diff
@@ -0,0 +1,178 @@
+    `variant` prop to specify the expander type.render
+
+    - 'readMore' expander with base-200 as its base color, and text-primary as the
+     text color. Used to get more information about an issue when giving your opinion
+     in the candidate opinions.
+
+    - 'category' expander with base-300 as its base color for the title, and base-100
+     for the content. Text is black and bolded. Used to open a summary about some
+     area.
+
+    - 'question' expander with base-100 as its base color. Text is black and bolded.render
+     Used to get more info about an already answeared question.render
+
+    - 'unansweared-question' expander with base-100 as its base color. Text is bolded and
+     colored as text-warning. Used to get more info about nonansweared questions.
+
+    ### Properties
+
+    - 'title': Title used for the expander. This is also used as the aria-label for 
+     the checkbox on which the expander operates on.
+    - 'variant': The type for the expander.
+    - 'iconColor': The color for the icon. Default color is primary.
+    - 'iconPos': The position for the icon. Default is text, which means the icon will
+        be where the text ends.
+    - 'customizeTitle': Variable to customize title.
+    - 'customizeContent': Variable to customize content.
+
+    You should not try to use a variant and customize at the same time.
+
+    ### Usage
+
+    ```tsx
+    <Expander variant="readMore" title="Example title">
+    <p> Example content <p/>
+    </Expander>
+
+    <Expander title="Example title" iconColor="primary" customizeTitle=" bg-base-100 text-primary"
+    customizeContent=" bg-base-300 text-info font-bold">
+    <p> Example content <p/>
+    </Expander>
+    ```
+   -->
+
+<div class={collapseClasses}>
+  <input type="checkbox" aria-label="open ${title}" on:click={rotateText} />
+  <div class={titleClasses}>
+    {title}
+    <div class="not-rotated-icon {iconIsRotated ? 'rotated-icon' : ''} ml-[0.4rem] {iconClass}">
+      <Icon name="next" size="sm" color={iconColor} aria-label="expander" />
+    </div>
+  </div>
+  <div class={`${contentClasses} ${iconIsRotated ? 'visible' : ''}`}>
+    <slot />
+  </div>
+</div>
+
+<style>
+  .not-rotated-icon {
+    --tw-rotate: 90deg;
+    transition: transform 0.2s linear;
+    transform: rotate(90deg);
+  }
+
+  .rotated-icon {
+    transform: rotate(270deg);
+  }
+
+  .collapse {
+    transition: none;
+  }
+
+  .collapse-title {
+    padding: 0.7rem;
+    align-items: center;
+    text-align: center;
+    width: 100%;
+    min-height: 0rem;
+  }
+
+  .display-flex {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .collapse-content {
+    display: flex;
+    text-align: center;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .collapse-content.visible {
+    padding: 0.7rem;
+  }
+
+  .collapse-title,
+  :where(.collapse > input[type='checkbox']) {
+    width: 100%;
+    min-height: 0rem;
+  }
```

Comment:
Maybe add a comment as to what this css does. Also, I don't think the styles need to be applied to the collapse-title class here because the same styles are already defined on lines 152 and 153.